include/ion/port.h

```diff
@@ -78,58 +79,87 @@ class Port {
      * @arg k: The key of the port which should be matched with BuildingBlock Input/Output name.
      * @arg t: The type of the value.
      */
-    Port(const std::string& n, Halide::Type t) : impl_(new Impl("", n, t, 0)), index_(-1) {}
+    Port(const std::string& n, Halide::Type t) : impl_(new Impl("", n, t, 0,  GraphID(""))), index_(-1) {}
```

Comment:
Is `GraphID("")` and `GraphID()` identical? Let's use either one if there is no meaning for these two use